spec/export/index.md

```diff
@@ -78,6 +84,24 @@ Export data from a FHIR server whether or not it is associated with a patient. T
 
   Specifies the format of the optional OperationOutcome response to the kick-off request. Currently, only application/fhir+json is supported.
```

Comment:
```suggestion
  Specifies the format of the optional OperationOutcome response to the kick-off request. Currently, only `application/fhir+json` is supported.
```

Reply:
@ricky1uphealth commit this change

Reply:
this had already been fixed https://github.com/1uphealth/bulk-data/blame/enhancements/spec/export/index.md#L91